printer.go

```diff
@@ -228,6 +231,10 @@ func (p *printer) printStruct() {
 				fieldName = field.Name
 			}
 
+			if p.exportedOnly && ('a' <= fieldName[0] && fieldName[0] <= 'z') {
```

Comment:
```suggestion
			if p.exportedOnly && field.PkgPath != "" {
```

A potentially better way to do this would be to see if the [StructField](https://pkg.go.dev/reflect#StructField).Pkg !=  "" instead of checking if it starts with a lower case letter